### PR TITLE
nrf_security: Updated documentation for nRF Security module

### PR DESCRIPTION
**Pull request by tejlmand**
This commit documents the advanced configuration section in nRF
Security.

Signed-off-by: Torsten Rasmussen <Torsten.Rasmussen@nordicsemi.no>

### CODE REVIEW

File: nrf_security/doc/nrf_security.rst [hunk 1]
```diff
@@ -669,15 +699,111 @@ Multiple Precision Integers (MPI) / Bignum calculation
...
+| Option                                         | Default | Min   | Max  |
++================================================+=========+=======+======+
+| :option:`CONFIG_MBEDTLS_MPI_WINDOW_SIZE`       | 6       | 1     | 6    |
++------------------------------------------------+---------+-------+------+
+| :option:`CONFIG_MBEDTLS_MPI_MAX_SIZE`          | 1024    | 0     | 1024 |
++------------------------------------------------+---------+-------+------+
+
+
+Elliptic Curves
+~~~~~~~~~~~~~~~
```
[the remark this answers is not in the document]
Comment (tejlmand) [reply nrf_security/doc/nrf_security.rst]: I know.
This is under advanced section, and the only part there we want to expose in the navigation pane is the `Advanced` section start.

File: crypto/doc/api.rst [hunk 2]
```diff
@@ -22,6 +22,41 @@ nRF CC310 platform library
...
+====================================
+
+.. doxygengroup:: nrf_cc310_platform_init
+   :project: nrfxlib
+   :members:
+
+CC310 Platform - Entropy APIs
+=============================
+
+.. doxygengroup:: nrf_cc310_platform_entropy
```
[the remark this answers is not in the document]
Comment (tejlmand) [reply crypto/doc/api.rst]: Will get that fixed.

**Comment by b-gent** [comment]
@tejlmand  I had a massive merge conflict and left some unnecessary to-be-squashed commits, sorry for the mess. The one that should be reviewed is of course 2a8a02 , please check my changes there and then squash as well.

File: crypto/doc/api.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (tejlmand) [reply crypto/doc/api.rst]: fixed

**Pull request merged**